apps/docs/spec/supabase_py_v2.yml

```
@@ -3654,7 +3789,12 @@ functions:
         name: Delete multiple records
         code: |
           ```python
-          response = supabase.table("countries").delete().in_("id", [1, 2, 3]).execute()
+          response = (
+              supabase.table("countries")
```

```suggestion
              supabase.table("characters")
```

lol this one was wrong originally

---

Looks like this didn't get changed before it got merged.

---

@charislam thanks a lot for looking into it, I was too fast to merge the change. I'll cross-check the data sources and submit in a new branch